platformio.ini

```diff
@@ -44,12 +44,15 @@ monitor_speed = 115200
 monitor_filters = esp8266_exception_decoder
 upload_port = 192.168.2.30
 upload_protocol = espota
+src_filter = +<*> -<*disabled*> -<httpledstripe_esp_refactored.*> -<httpledstripe_esp.cpp>
 build_flags = 
     ${common_env_data.build_flags}
     -DLED_COUNT_TOT=250
     -DLED_OFFSET=0
     ;-DHAS_KNOB_CONTROL
-lib_deps = ${common_env_data.lib_deps_builtin}
```

Comment:
why did you remove the common_env_data.lib_deps_builtin? please revert to this generic apprach